zokrates_core/src/proof_system/bn128/gm17.rs

```diff
@@ -16,26 +21,46 @@ impl GM17 {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+struct GM17ProofPoints {
+    a: G1PairingPoint,
+    b: G2PairingPoint,
+    c: G1PairingPoint,
+}
+
+impl Proof<GM17ProofPoints> {
+    fn from_json(str: &str) -> Self {
+        serde_json::from_str(str).unwrap()
```

Comment:
Just making sure we're not unwrapping on a user error here?

Reply:
Moved this method to https://github.com/Zokrates/ZoKrates/pull/583/files#diff-189d3ba6c93acd77029cea5c8a6a386aR26